app/web/markdown/faq.md

```diff
@@ -7,41 +7,41 @@ type: general
+Yes. We not only strive to be a viable alternative to Couchsurfing™ but also to be the most safe, modern, and active couch surfing community. We aim to build the features people have come to love across all platforms, but with many improvements. We have also added completely new features which help enhance the couch surfing experience for everyone.
 
 ### Is there going to be a mobile app for Couchers.org on both iOS and Android?
 
-Yes, mobile apps are critical for all travel platforms. We are currently working on our [web app](https://couchers.org), but will be creating native apps for iOS and Android as we progress.
+Yes, mobile apps are critical for all travel platforms. We are currently working on native apps for iOS and Android and hope to release them in early 2026. For now, you can [bookmark the Couchers.org website to your phone's home screen](https://couchers.org/blog/2025/05/10/how-to-add-couchers-org-phone-home-screen) to have it behave like an app.
 
 ### How will you prevent this platform from ever becoming a for-profit like Couchsurfing&#8482; did?
 
 We fundamentally believe that [attempting to make a profit out of couch surfing is a bad idea](/issues/profit-and-incentives). It introduces incentives that damage the community and would not make financial sense — the couch surfing idea, based on non-transactional experiences, is not monetizable. This is about societal value, not monetary value.
 
-We are keeping the platform as a non-profit forever. Our plan to follow this relies on three fundamental pillars:
+We are keeping the platform as a non-profit **forever**. Our plan to follow this relies on three fundamental pillars:
 
 1. We are legally established as a **non-profit foundation**, and [our](/foundation) constitution contains provisions that prevents the company from ceasing to be a non-profit, or transferring its assets to an entity that is not a non-profit.
 
 2. We will carry out a policy of **distributed moderation**, so that we will engage hundreds of moderators as volunteers around the world to moderate their own communities. We will make the platform reliant on volunteers, and so the entity controlling the platform could not be a for-profit business without violating laws in many countries. The foundation would have to remain as a non-profit to continue operating.
 
-3. Our code base is **open source** and anybody can spin up an alternative instance. If the community ever comes to feel that the leaders of the platform are not acting in their interest, they can simply fork the codebase, making a copy that is under control of new management.
+3. Our code base is [**open source**](https://couchers.org/open-source) and anybody can spin up an alternative instance. If the community ever comes to feel that the leaders of the platform are not acting in their interest, they can simply fork the codebase, making a copy that is under control of new management.
```

Comment:
I don't think the link here is the right place to link to, given it's not much about our OSS stuff, but rather about how we use OSS. Can either change the text to say we embrace open source (but that's not super relevant in this point), or link e.g. to github?